main.go

```diff
@@ -48,7 +48,7 @@ func parse(in []byte) ([]byte, error) {
 		return nil, err
 	}
 
-	data, ok := s["data"].(map[string]string)
```

Comment:
Here if type can not be converted to `map[string]string` then it goes to line #53

Reply:
If we are doing something like `kubectl get secret admin1 -o yaml` and there (for example) is a token inside then it will return it AS IS without decoding. Basically it works only for ligin/pass.

Reply:
thanks!